### ISSUE TITLE
Question: Is there an out-of-box way to use "no" instead of "zero" when using ToQuantity?

### ISSUE THREAD

**Issue by dealdiane** [Humanizr/Humanizer#580]
Is there a way so that calling `$"You have {"item".ToQuantity(0, ShowQuantityAs.Words)}"` would return "You have **no** items" instead of "You have **zero** items"?

**Comment by aloisdg** [comment]
Just as a small hint, I think interpolation string here is a bit useless.

**Comment by dealdiane** [comment]
Yeah, I agree. The only reason for that is because I personally find it simpler to read/represent (here in github) what I'm trying to do than using `+`.

**Comment by clairernovotny** [comment]
Thank you for the question. There is no built-in special replacement of zero with “no”, because that wording is application- and locale-specific. The existing composition point is `"item".ToQuantity(0, ShowQuantityAs.None)`, which returns the correctly inflected noun (`"items"`); the caller can then prefix its chosen localized word, for example `"no " + ...`. Since the current API already provides the inflection without forcing a zero word, I’m closing this as answered/not planned.
